Classes/Service/StatisticsService.php

```diff
@@ -190,7 +191,7 @@ public function getApplications($dates)
                 ) c",
                 "1 = 1"
             );
-        return $applications;
+        return ['value' => $applications, 'total' => intval($applications["men"])+intval($applications["women"])];
```

Comment:
You don't need `intval` if you are performing a numeric operation like `+`. Also please use an `(int)$value` cast instead.